app/section/sections/browser-mobile-first/ui.test.js

```diff
@@ -0,0 +1,189 @@
+
+  await t
+    .expect(submitButton.exists).ok()
+    .click(submitButton)
+    .expect(getLocation()).contains('/S100000-001/dashboard/browser-based');
+});
+
+test('should populate the questions with existing data', async (t) => {
+  pageSetup(t, 200, browserMobileFirstMarketingData);
+
+  const pluginsRequiredQuestion = Selector('[data-test-id="question-mobile-first-design"]');
+  const yesRadiobutton = pluginsRequiredQuestion.find('.nhsuk-radios__item:nth-child(1)');
+  const noRadiobutton = pluginsRequiredQuestion.find('.nhsuk-radios__item:nth-child(2)');
+
+  await t
+    .expect(yesRadiobutton.find('input:checked').exists).ok()
+    .expect(noRadiobutton.find('input:checked').exists).notOk();
+});
+
+test('should show error summary and validation for mobile first design question indicating it is mandatory', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-mobile-first')
+    .reply(400, {
+      required: ['mobile-first-design'],
+    });
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const mobileFirstDesignQuestion = Selector('[data-test-id="question-mobile-first-design"]');
+  const submitButton = Selector('[data-test-id="section-submit-button"]');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .click(submitButton.find('button'))
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummaryList.find('li').count).eql(1)
+    .expect(errorSummaryList.find('li:nth-child(1)').innerText).eql('Please select whether or not this solution was designed with a mobile first approach')
+    .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#mobile-first-design')
+
+    .expect(mobileFirstDesignQuestion.find('[data-test-id="radiobutton-options-error"]').exists).ok()
+    .expect(mobileFirstDesignQuestion.find('.nhsuk-error-message').innerText).eql('Error:\nPlease select whether or not this solution was designed with a mobile first approach');
+});
+
+test('should goto anchor when clicking the mobile first design summary error link', async (t) => {
+  await pageSetup(t);
+
+  nock('http://localhost:8080')
+    .put('/api/v1/Solutions/S100000-001/sections/browser-mobile-first')
+    .reply(400, {
+      required: ['mobile-first-design'],
+    });
+
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const errorSummaryList = Selector('.nhsuk-error-summary__list');
+  const submitButton = Selector('[data-test-id="section-submit-button"]');
+
+  const getLocation = ClientFunction(() => document.location.href);
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .click(submitButton.find('button'))
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummaryList.find('li:nth-child(1) a').count).eql(1)
+    .expect(errorSummaryList.find('li:nth-child(1) a').getAttribute('href')).eql('#mobile-first-design')
```

Comment:
These assertions we have already made in the test above. Just test the anchor behavior